spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -537,7 +537,7 @@ definitions:
            size_fn: n_sats
            desc: Array of STEC polynomial coefficients and its bounds for each space vehicle.
 
- - MSG_SSR_TILE_DEFINITION_DEP:
+ - MSG_SSR_TILE_DEFINITION_DEP_A:
```

Comment:
@silverjam I'm a bit uncertain about this change. Since this message now will consist of two deprecated versions, I added a suffix to the previous deprecated version but then that name doesn't exist anymore, which may create a compatibility issue? Maybe it's more correct to leave the first deprecated message unchanged and then add a suffix to the current tile definition so the message versions become:
```
MSG_SSR_TILE_DEFINITION_DEP
MSG_SSR_TILE_DEFINITION_DEP_A
MSG_SSR_TILE_DEFINITION
```
instead of 
```
MSG_SSR_TILE_DEFINITION_DEP_A
MSG_SSR_TILE_DEFINITION_DEP_B
MSG_SSR_TILE_DEFINITION
```

Reply:
@Davinco Either way is fine, we should go with whatever precedent we already have with existing messages, if there isn't a consistent precedent then we can pick whatever makes the most logic sense.  The main backwards compatibility that we want to guard against is wire/format compatibility, backwards compat with compilation is nice, but not strictly necessary.  Wire compatibility means: the size and format of bytes in the message cannot change, for bit-field messages we can add new bit-fields and deprecate old ones and in some cases we can change the interpretation of bytes (e.g. from unsignged to signed) but these cases need to be considered carefully.

Reply:
nice, thanks for the clarification :raised_hands: